src/routes.ts

```diff
@@ -53,6 +54,22 @@ export default [
       ]
     }
   },
+  {
+    path: '/amalgamation-selection',
+    name: Routes.AMALGAMATION_SELECTION,
+    component: AmalgamationSelection,
+    meta: {
+      requiresAuth: true,
+      breadcrumb: [
+        {
+          text: 'Amalgamation Selection',
+          disabled: false,
+          exact: true,
+          to: { name: Routes.AMALGAMATION_SELECTION }
+        }
+      ]
+    }
+  },
```

Comment:
It makes sense to me to update the breadcrumb when they're on this panel (view) 👍